test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -5734,7 +5734,7 @@ public void TestCrossValidationMacroWithStratification()
                 getter(ref stdev);
                 foldGetter(ref fold);
                 Assert.True(ReadOnlyMemoryUtils.EqualsStr("Standard Deviation", fold));
-                Assert.Equal(0.0087, stdev, 5);
+                Assert.Equal(0.02582, stdev, 5);
```

Comment:
This changed because of `ensureZeroUntouched = false` (see [https://github.com/dotnet/machinelearning/pull/5227#discussion_r441994977](https://github.com/dotnet/machinelearning/pull/5227#discussion_r441994977) )